objectRecognitionSource/src/main/java/io/vantiq/extsrc/objectRecognition/ObjectRecognitionConfigHandler.java

```diff
@@ -303,8 +303,20 @@ public void run() {
             }
         };
         
-        // Start polling if pollRate is non-negative
-        if (general.get("pollRate") instanceof Integer) {
+        // Start polling if pollTime is non-negative
+        if (general.get("pollTime") instanceof Integer) {
+            polling = (Integer) general.get("pollTime");
+            if (polling > 0) {
+                int pollRate = polling;
+                source.pollTimer = new Timer("dataCapture");
+                source.pollTimer.schedule(task, 0, pollRate);
+            } else if (polling == 0) {
+                source.pollTimer = new Timer("dataCapture");
+                source.pollTimer.scheduleAtFixedRate(task, 0, 1);
+                // 1 ms will be fast enough unless image gathering, image processing, and data sending combined are
+                // sub millisecond
+            }
+        } else if (general.get("pollRate") instanceof Integer) {
```

Comment:
Might be good here to put out a warning that `pollRate` is deprecated & should be replaced by `pollTime`.

Reply:
Should I add this as a log.trace or .debug, or something different?

Reply:
I would do it as a `log.warn()` -- it's something we want them to do. The code is being helpful about backwards compatibility, but we'd prefer that they change ASAP.  When `pollTime` is used, there will be no output...